tools/federation/src/main/java/org/eclipse/rdf4j/federated/repository/FedXRepositoryConfig.java

```diff
@@ -187,6 +203,14 @@ public void parse(Model m, Resource implNode) throws RepositoryConfigException {
 			Models.objectLiteral(m.getStatements(implNode, DATA_CONFIG, null))
 					.ifPresent(value -> setDataConfig(value.stringValue()));
 
+			Models.objectResource(m.getStatements(implNode, FEDX_CONFIG, null))
+					.ifPresent(res -> {
+						if (getConfig() == null) {
```

Comment:
Note: I chose to do this so the previous behaviour is preserved:
1. If no `FedXConfig` has been provided (programmatically or via template) then it will be `null`
2. When exporting, `FedXConfig` is not included (when `null`) rather than being expanded into defaults (see `export` above)

Reply:
Good considerations! Please see my comments for the exporting

Reply:
Changed with the other suggestion: 5a160dfd1bfcaa48e3c5